ci/pipeline.yml

```
@@ -222,6 +222,29 @@ jobs:
         params:
           BASE_STEMCELL: ubuntu-noble
 
+  - name: test-acceptance-ubuntu-resolute
+    public: true
+    serial: true
+    plan:
+      - in_parallel:
+          - get: bosh-dns-release
+            passed: [ pre-integration-fan-in ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)